index.html

```diff
@@ -29,6 +29,14 @@ <h2>Experience</h2>
             </ul>
         </section>
 
+        <section id="skills">
+            <h2>Skills</h2>
+            <ul>
```

Comment:
nice job, have you tried the foo function from the overhead of adding types to your code as well